cl/.gitignore

```diff
@@ -0,0 +1,246 @@
+
```

Comment:
I'd probably just put this in the root if we need it, rather than having the complexity of multiple ignore files here and there, unless there's something we gain from the way you set it up.

Reply:
Good call. Overlooked that, but I definitely agree.